src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -159,6 +159,9 @@
     <Compile Include="SQL\SqlBulkCopyTest\DataConversionErrorMessageTest.cs" />
     <Compile Include="SQL\SqlBulkCopyTest\CopyWidenNullInexactNumerics.cs" />
   </ItemGroup>
+  <ItemGroup Condition="$(OS)=='Windows' AND ('$(TestSet)' == '' OR '$(TestSet)' == '2')">
```

Comment:
According to [this](https://github.com/dotnet/msbuild/issues/2468), `$(OS)` will be `Windows_NT`. It should be `'$(TargetsWindows)' == 'true'`, which is defined in `test/Directory.Build.Props` and already being used in another condition